### PR TITLE
https://issues.redhat.com/browse/ACM-8363 updates

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8363

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

File: clusters/install_upgrade/install_connected.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ The {mce-short} is installed with Operator Lifecycle Manager, which manages the
 
 **Important:**
```
Comment (dockerymick) [inline clusters/install_upgrade/install_connected.adoc]:
```suggestion
*Important:* 
```

**Review by dislbenn — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the reviews and approvals! @cadawson @dislbenn @mihirlele  
Merging this PR

**Pull request merged**